.travis.yml

```diff
@@ -76,25 +76,25 @@ jobs:
     # main profile
     - stage: compile
       script: ./tools/travis_controller.sh
-      env: PROFILE="-Dhadoop.version=2.8.3 -Pinclude-kinesis -Dinclude_hadoop_aws"
+      env: PROFILE="-Dhadoop.version=2.8.3 -Pinclude-kinesis -Dinclude_hadoop_aws -Dscala-212"
```

Comment:
This switches all profiles to Scala 2.12 - I guess we should only switch half of them, or make Scala 2.12 a nightly cron-job profile?

Reply:
this is just for testing purposes and was not intended to be merged. Scala 2.12 will be covered with a nightly cron-job as you suggested.